cloudera-on-centos/azuredeploy.json

```diff
@@ -28,10 +28,10 @@
         "description": "password for the Cloudera Manager (A mix of upper and lower-case characters, digits and symbols)"
       }
     },
-    "storageAccountPrefix": {
+    "storageAccountSuffix": {
```

Comment:
can we just move these to a variable and generate a name - not sure the actual uri is important for this scenario...